docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/hpc-launching-architecture.rst

```diff
@@ -190,6 +190,9 @@ provided that they do not conflict with Determined-controlled settings.
 +------------------------+-------------------------------------------------------------------------------------------+
 | ``--wckey``            | A value identified by the ``resource_manager.job_project_source`` configuration.          |
 +------------------------+-------------------------------------------------------------------------------------------+
+| ``--no-requeue``       | Disable any potential automatic requeue of the job by SLURM. Determined will handle the   |
```

Comment:
Is it that Determined will supply this option automatically? If so does it really matter if the user specifies it or not?

Reply:
This table is about:

Common Slurm Options
The following options are generated by Determined and *cannot be directly specified by the user.* Additional options specified by slurm.sbatch_args will be added to the generated batch file provided that they do not conflict with Determined-controlled settings.

We should also prevent it from being specified in our validation.

Reply:
Or more specifically, we should prevent the user from requesting --requeue.

Reply:
This table lists the options provided by the determined by default. And it is mentioned in the preceding paragraph that the user options that conflict with these options (in our case --requeue) will be ignored.